rest-proxy-boot/build.gradle

```diff
@@ -10,7 +10,8 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath("org.springframework.boot:spring-boot-gradle-plugin:1.3.6.RELEASE")
+        classpath("org.springframework.boot:spring-boot-gradle-plugin:1.3.7.RELEASE")
+        classpath "io.spring.gradle:dependency-management-plugin:0.6.1.RELEASE"
```

Comment:
What does this `classpath "io.spring.gradle:dependency-management-plugin:0.6.1.RELEASE"` line do?